### ISSUE TITLE
How to dim or change color of a switch?

### ISSUE THREAD

**Issue by hkoren** [dpjanes/iotdb-smartthings#12]
I tried setting level=....   but that just turns off the switch

**Comment by dpjanes** [comment]
I only developed for what I have. The code's quite easy to modify, if you want to fork and make a patch to add new features.

**Comment by hkoren** [comment]
Wish I knew the modifications to make....  This seems to indicate some dimmer functionality is available but I can't seem how to indicate that my switch is a dimmer switch https://iotdb.org/iotdb/models/light-dimmer

**Comment by dpjanes** [comment]
Switches can be on or off, that's what they do. Can you give more information about _your_ smartthings set, what type of switch it is, etc.. I'm interested in helping, but I can't guess what your setup is.

**Comment by hkoren** [comment]
I'm using a GoControl Z-Wave Plug-in Dimmer Module.  Looking through the code and docs it seems like I will need to add capability.switchLevel  to the IOTDB.bridge smartapp for this control to be possible.

**Comment by dpjanes** [comment]
Yes, that sounds about right. The pattern for adding this to the code is quite simple.
